.travis.yml

```diff
@@ -1,23 +1,29 @@
 language: python
+python: 3.5
 cache:
   directories:
     - $HOME/virtualenv/python3.5.3/
-env:
-  - TEST_SUITE=unit
-  - TEST_SUITE=journey
-python: 3.5
+
+jobs:
+  include:
+    - stage: unit
+      script: coverage run -m unittest discover tests/unit
+
+    - stage: journey
+      script: coverage run -m unittest discover tests/journey
+
+    - stage: deploy
+      script: skip
```

Comment:
is this really necessary to be explicit? I thought not adding any would automagically skip the scripting step if not stated.

Reply:
When you do I'll merge this =)

Reply:
I thought that I already had answered it but this line is necessary based on [this piece of build job documentation](https://docs.travis-ci.com/user/build-stages/#Build-Stages-and-Deployments) because each step try to run a script but it's not necessary for a deploy job.